client/editEvents.js

```diff
@@ -201,10 +201,10 @@ Template.editEvents.helpers({
     return startDate.fromNow() + " (" + rangeStr + ")";
   },
   roundOpen: function() {
-    return this.status == wca.roundStatuses.open;
+    return this.isOpen();
```

Comment:
`roundOpen` was never used, so I killed that.
`canCloseRound` had an off-by-one boolean error after sloppy refactoring by me. Fixed.

But I like the pattern of `{{#if canOpenRound}}` /  `{{#if canCloseRound}}` / `{{#if canAdvanceRound}}`, and prefer to keep that, even though one of them can be inlined with the current implementation.